src/benchmarks/micro/Serializers/Xml_ToStream.cs

```diff
@@ -52,9 +54,23 @@ public void DataContractSerializer_()
             dataContractSerializer.WriteObject(memoryStream, value);
         }
 
+        [BenchmarkCategory(Categories.Libraries)]
+        [Benchmark(Description = nameof(XmlDictionaryWriter))]
+        public void DataContractSerializer_BinaryXml_()
+        {
+            memoryStream.Position = 0;
+            ((IXmlBinaryWriterInitializer)xmlDictionaryWriter).SetOutput(memoryStream, null, null, ownsStream: false);
```

Comment:
>xmlDictionaryWriter).SetOutput [](http://example.com/codeflow?start=42&length=30)

Again, consider using [IterationSetup] for this.

Reply:
That can be removed, but then it will emit slight different output (no start of document) for most calls.
I did that change int the separate branche mentioned above